Formula/jsign.rb

```diff
@@ -0,0 +1,50 @@
+class Jsign < Formula
+  desc "Tool for signing Windows executable files, installers and scripts"
+  homepage "https://ebourg.github.io/jsign/"
+  url "https://github.com/ebourg/jsign/archive/refs/tags/5.0.tar.gz"
+  sha256 "7b77a12aaea4f404e7b243bd58cfde485eb03b44219e128338c9fe6617ad1fa1"
+  license "Apache-2.0"
+  head "https://github.com/ebourg/jsign.git", branch: "master"
+
+  depends_on "maven" => :build
+  depends_on "openjdk@17" # The build fails with more recent JDKs
+
+  def install
+    ENV["JAVA_HOME"] = Formula["openjdk@17"].opt_prefix
+    system("mvn", "--batch-mode", "package", "--projects", "jsign-core,jsign-cli,jsign-ant,jsign", "-DskipTests",
+           "-Djdeb.skip", "-Dmaven.javadoc.skip")
+
+    # Fetch the version from the pom (required to build from HEAD)
+    require "rexml/document"
+    pom = REXML::Document.new(File.new("pom.xml"))
+    version = REXML::XPath.first(pom, "string(/pom:project/pom:version)", "pom" => "http://maven.apache.org/POM/4.0.0")
+
+    libexec.install Dir["jsign/target/jsign-#{version}.jar"]
+    bin.write_jar_script libexec/"jsign-#{version}.jar", "jsign", "-Djava.net.useSystemProxies=true -Dbasename=jsign -Dlog4j2.loggerContextFactory=net.jsign.log4j.simple.SimpleLoggerContextFactory"
+    bash_completion.install "jsign/src/deb/data/usr/share/bash-completion/completions/jsign" => "jsign"
+    man1.install "jsign/src/deb/data/usr/share/man/man1/jsign.1"
+  end
+
+  test do
+    resource "testdata" do
+      url "https://github.com/ebourg/jsign/archive/refs/tags/5.0.tar.gz"
+      sha256 "7b77a12aaea4f404e7b243bd58cfde485eb03b44219e128338c9fe6617ad1fa1"
+    end
```

Comment:
Is this just an exact copy of the source tarball? Which files are needed from it?

Reply:
Yes that's the source tarball. The test resources from the `jsign-core/src/test/resources` directory are reused.

Reply:
Let's just do
```ruby
pkgshare.install "jsign-core/src/test/resources"
```
in the `install` method, please. This directory isn't very large (~400KB), so it doesn't hurt to carry it around. You can delete the files you don't need first if you prefer.

Reply:
Ok but I find it a bit odd to have that installed on the user's system. With the source tarball being reused nothing is downloaded by homebrew nor duplicated on the disk, the resource is symlinked to the source tarball already cached, that really sounds optimal, but I may be missing something.

Reply:
Understood. Is it any better if the few files used are downloaded directly from GitHub? That's about 7 HTTP requests for less than 100KB.

Reply:
`pkgshare.install` is now used